### PR TITLE
Settings page with new design system

### PR DESCRIPTION
**Pull request by ainouzgali**

### CODE REVIEW

**Review by scopsy — COMMENTED** [review]
Looks fantastic!! A few small comments, let me know what you think :)

File: apps/web/src/pages/settings/components/BrandingForm.tsx [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+
+export const dropzoneChildren = (status: DropzoneStatus, image) => (
+  <Group position="center" spacing="xl" style={{ minHeight: 100, minWidth: 100, pointerEvents: 'none' }}>
+    {!image ? (
+      <Upload style={{ width: 80, height: 80, color: colors.B60 }} />
+    ) : (
+      <img src={image} style={{ width: '100%', height: 80 }} alt="avatar" />
+    )}
+  </Group>
+);
```
[the remark this answers is not in the document]
Comment (ainouzgali) [reply apps/web/src/pages/settings/components/BrandingForm.tsx:40]: Makes perfect sense, thanks!

File: apps/web/cypress/tests/settings.spec.ts [hunk 2]
```diff
@@ -31,24 +30,24 @@ describe.skip('Settings Screen', function () {
...
     cy.reload();
 
-    cy.get('.ant-tabs-tab-btn').contains('Email settings').click();
+    cy.get('.mantine-Tabs-tabsList').contains('Email Settings').click();
     cy.getByTestId('sender-email').should('have.value', 'new-testing@email.com');
     cy.getByTestId('sender-name').should('have.value', 'Test Sender Name');
   });
 
-  it('should update logo', function () {
+  it.skip('should update logo', function () {
```
[the remark this answers is not in the document]
Comment (ainouzgali) [reply apps/web/cypress/tests/settings.spec.ts:50]: yes, it did not work for me in the old page as well. Probably access issues. Also, the tests for the color input, I still need to work on translating it from ant, it is not as trivial as the others. @scopsy , Maybe we could do it together.

File: apps/web/src/pages/settings/WidgetSettingsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+      ),
+    },
+  ];
+
+  return (
+    <PageContainer>
+      <PageHeader title="Settings" />
+      <Container fluid mt={15} ml={5}>
+        <Tabs active={activeTab} onTabChange={setActiveTab} menuTabs={menuTabs} />
+      </Container>
```
[the remark this answers is not in the document]
Comment (ainouzgali) [reply apps/web/src/pages/settings/WidgetSettingsPage.tsx:115]: Sure :)

File: apps/web/cypress/tests/settings.spec.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (scopsy) [reply apps/web/cypress/tests/settings.spec.ts:50]: Sure! Would love to :)

**Review by scopsy — APPROVED** [review]
Looks great!

**Pull request merged**